docs/upgrade/upgrade-to-enterprise/upgrade-guide-from-5.0-to-2022.1/metric-update-5.0-to-2022.1.rst

```diff
@@ -0,0 +1,86 @@
+=================================================================
+ScyllaDB Metric Update - Scylla 5.0 to Scylla Enterprise 2022.1
+=================================================================
+
+New Metrics
+------------
```

Comment:
AFAIK, there are **no** metric updates from 5.0 to 2022.1
@amnonh please advise

Reply:
I couldn't find any updates

Reply:
@tzach I've updated the page about metrics.